nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -512,6 +513,14 @@ def _validate_and_override_config(self):
                 'Make sure the number of model chunks is the same across all pipeline stages.'
             )
 
+        if self.cfg.get('ub_tp_comm_overlap', False):
+            if not self.cfg.get('transformer_engine', False) or not self.cfg.get('sequence_parallel', False):
+                logging.info(
+                    "Userbuffer tensor-parallel communication overlap is available with both Transformer Engine and sequence-parallelism."
+                )
```

Comment:
this should be `.warning` and maybe add ".. only available with ... Setting ub_tp_comm_overlap to True"